cli/command_server_start.go

```diff
@@ -273,6 +274,15 @@ func (c *commandServerStart) run(ctx context.Context) error {
 	return errors.Wrap(srv.SetRepository(ctx, nil), "error setting active repository")
 }
 
+// shutdownServer shutdown http server and close the repository.
+func shutdownServer(ctx context.Context, httpServer *http.Server) {
+	log(ctx).Infof("Shutting down...")
+
+	if serr := httpServer.Shutdown(ctx); serr != nil {
+		log(ctx).Debugf("unable to shut down http server: %v", serr)
+	}
+}
+
```

Comment:
@aaron-kasten do you need this function for `onRepositoryFatalError` ? Can we split apart the `onRepositoryFatalError` trigger to a separate PR or is that required for this PR ?

Reply:
@julio-lopez @Shrekster yes... this can easily be put into a separate PR.  This is really here for better logging/notification in the logs. (I have a strong preference to capture all errors and log where appropriate).